packages/debian/entrypoint.sh

```diff
@@ -35,6 +35,22 @@ rpm2deb() {
     chown "${TARGET_UID}:${TARGET_GID}" * 
 }
 
+
+download_packages() {
+#<Package_name>=<Package_version>
+for pkg in $(cat list.txt)
+do
+        echo "$pkg" >> packages_list.txt
+        for dep in $(apt-rdepends "$pkg" | grep -E 'Depends|Recommends|Suggests' | cut -d " " -f4)
+                do echo "$dep" >> packages_list.txt
+        done
+done
+for todl in $(sort -u packages_list.txt)
```

Comment:
I'm not sure that's the way to read a file line by line, my dusted memories about Bash reminds me of something like

```sh
while IFS= read -r line
do
  echo "$line"
done < "packages_list.txt"
```

TBC.